source/basic.tex

```diff
@@ -1777,12 +1777,13 @@
 such a name undergoes unqualified name lookup from the point where it appears.
 
 \pnum
-An unqualified name that is a component name\iref{expr.prim.id.unqual} of
+If a \grammarterm{conversion-function-id} is a qualified name
+with lookup context $S$\iref{basic.lookup.qual},
+an unqualified name $U$ that is a component name\iref{expr.prim.id.unqual} of
 a \grammarterm{type-specifier} or \grammarterm{ptr-operator} of
-a \grammarterm{conversion-type-id} is looked up in the same fashion
-as the \grammarterm{conversion-function-id} in which it appears.
-If that lookup finds nothing, it undergoes unqualified name lookup;
-in each case, only names
+its \grammarterm{conversion-type-id} is looked up in $S$;
```

Comment:
"is looked up in" is imprecise.  The formal equivalent to the previous would be to say "undergoes qualified lookup in _S_", but since none of the embellishments of qualified lookup can apply here we can follow the next paragraph and say

> …, lookup for an unqualified name that is a component name … performs a search in the scope associated with _S_.  Otherwise, or if that lookup finds nothing, it undergoes unqualified name lookup.  In each case, …

It might be profitable to first establish the "only names that denote" rule:

> Lookup for an unqualified name _U_ that is a component name of a *type-specifier* or *ptr-operator* of a *conversion-type-id* _T_ considers only names….  If _T_ appears in a *conversion-function-id* that is a qualified name…, lookup for _U_ performs…; if that lookup finds nothing, ….

If someone were feeling ambitious, they could actually combine the paragraphs by establishing two circumstances where a search happens with unqualified lookup as a backup, although you still need the restriction to types for this case.